### DISCUSSION TITLE
How to extract time stamps from audio?

### DISCUSSION THREAD

**Discussion by neeraj-mahapatra** [openai/whisper#1611] [Q&A]
I have a 30-second audio file. I am able to transcribe it easily using 

```
import openai
audio_file= open(r"path\audio.mp3", "rb") #path_to_audio_file
transcript = openai.Audio.transcribe("whisper-1", audio_file)
```

It gives text as output without time stamps, but I want output with time stamps.
Is it possible using whisper. If yes how? Is there any other ways to do it too?

**Comment by phineas-pta** [comment]
your question should be asked at: https://community.openai.com/tag/whisper
